### DISCUSSION TITLE
Welcome to tdm-gcc Discussions!

### DISCUSSION THREAD

**Discussion by jmeubank** [jmeubank/tdm-gcc#29] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by BAW2501** [comment]
hey I was wondering if your installer had a silent install command or such?
I'm attempting to add TDM-GCC to the windows package manager so it would be more convenient to install but it seems the installer fails to install probably cause of the above  ,you can take a look at the pull request for it in winget-pkgs repository if you wish (search jmeubank.TDM-GCC)
